Power management (#24)
hostmobility/documentation: docs/system/mx4/wake_up_cause.md

---
title: Wake up cause
tags:
  - Wake up cause
  - Power Management
  - MX-4
---

> Here is a list of possible wake up signals it is used both for enable the wake up mode and to read what mode it was that woke the system.

Reviewer: ```suggestion
Here is a list of possible wake-up signals used both to enable the wake up mode and to read which of the modes that woke up the system.
```

## Read
```bash
MX4_SPI_DIR=/opt/hm/pic_attributes
cat $MX4_SPI_DIR/ctrl_wakeup_cause
#or
/opt/hm/wake_up_cause.sh
```

> ## activate coprocessor wake up source

Reviewer: ```suggestion
## Enable coprocessor wake up source
```
